From: Hannes Reinecke <hare@suse.de>
Date: Thu, 18 Aug 2016 13:09:25 +0200
Subject: libfc: fixup RTV handling
References: bsc#990245
Git-commit: 7c5a51b8f82fcfba1925fa64f08413c8258590d2
Patch-mainline: v4.10-rc1

A missing case label was causing the RTV handler never to be called,
and always resulting in an RTV reject.

Signed-off-by: Hannes Reinecke <hare@suse.com>
---
 drivers/scsi/libfc/fc_rport.c |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

--- a/drivers/scsi/libfc/fc_rport.c
+++ b/drivers/scsi/libfc/fc_rport.c
@@ -1268,7 +1268,7 @@ static void fc_rport_enter_prli(struct f
 }
 
 /**
- * fc_rport_els_rtv_resp() - Handler for Request Timeout Value (RTV) responses
+ * fc_rport_rtv_resp() - Handler for Request Timeout Value (RTV) responses
  * @sp:	       The sequence the RTV was on
  * @fp:	       The RTV response frame
  * @rdata_arg: The remote port that sent the RTV response
@@ -1766,6 +1766,7 @@ static void fc_rport_recv_req(struct fc_
 	case ELS_RRQ:
 	case ELS_REC:
 	case ELS_RLS:
+	case ELS_RTV:
 		fc_rport_recv_els_req(lport, fp);
 		break;
 	default:
